runtime-extension-model/src/main/java/org/mule/runtime/core/api/extension/provider/MuleExtensionModelDeclarer.java

```diff
@@ -691,6 +701,9 @@ private void declareFlow(ExtensionDeclarer extensionDeclarer) {
         .ofType(BASE_TYPE_BUILDER.numberType().integer().range(1, null).build())
         .withExpressionSupport(NOT_SUPPORTED);
 
+    addBusinessEventParameters(flow,
```

Comment:
I will approve this anyway, but my suggestion would be to rename this method to something referencing the tracking module. Although the parameter is called business events, that's a user facing name. From the POV of a developer onboarding into the code base, this is much clearer if it refers to the actual module name and namespace that it represents